routes/clams.js

```diff
@@ -0,0 +1,136 @@
+const express = require('express');
+const { pool } = require('../server/db');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+
+// Get data for all clams
+router.get('/', async (req, res) => {
+  try {
+    const clams = await pool.query('SELECT * FROM clams;');
+    res.status(200).json(keysToCamel(clams));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for specific clam based on clamId
+router.get('/:clamId', async (req, res) => {
+  try {
+    const clam = await pool.query('SELECT * FROM clams WHERE clamId = $(clamId);');
+    res.status(200).json(keysToCamel(clam.rows));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for clams based on rakerId
+router.get('/rakers/:rakerId/clams', async (req, res) => {
+  try {
+    const clam = await pool.query('SELECT * FROM clams WHERE rakerId = $(rakerId)');
+    res.status(200).json(keysToCamel(clam.rows));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Create clam
+router.post('/', async (req, res) => {
+  try {
+    const { clamId, rakerId, lat, long, length, width, weight, comments, image } = req.body;
+    const clam = await pool.query(
+      `INSERT INTO clams (
```

Comment:
there's no sign of the clam attributes being passed into the query here; see my comment a little above on how to best fix it

tldr: more testing